shopinvader/models/res_partner.py

```diff
@@ -21,15 +33,129 @@ class ResPartner(models.Model):
     )
     # In europe we use more the opt_in
     opt_in = fields.Boolean(compute="_compute_opt_in", inverse="_inverse_opt_in")
-    shopinvader_enabled = fields.Boolean(
+    is_shopinvader_active = fields.Boolean(
         string="Shop enabled",
-        default=True,
-        help="Enable user account on the frontend. "
-        "If this is disabled the user cannot log in on all the websites. "
-        "This behavior must be enabled on the backend "
-        "via 'Validate customers' flag.",
+        help="This address is enabled to be used for Shopinvader.",
+    )
+    parent_has_shopinvader_user = fields.Boolean(
+        help="This partner belongs to Shopinvader user.",
+        compute="_compute_parent_has_shopinvader_user",
+        compute_sudo=True,
```

Comment:
```suggestion
        related="parent_id.has_shopinvader_user",
```

why not using a related field here?